config/initializers/training_routes.rb

```diff
@@ -657,7 +663,215 @@
+    amount: 25_000,
+    allocation_subjects: [
+      AllocationSubjects::BIOLOGY,
+      AllocationSubjects::DESIGN_AND_TECHNOLOGY,
+      AllocationSubjects::GEOGRAPHY,
+      AllocationSubjects::MODERN_LANGUAGES,
+      AllocationSubjects::ANCIENT_LANGUAGES,
+    ],
+  ),
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:provider_led_postgrad],
+    amount: 10_000,
+    allocation_subjects: [
+      AllocationSubjects::ART_AND_DESIGN,
+      AllocationSubjects::ENGLISH,
+      AllocationSubjects::MUSIC,
+      AllocationSubjects::RELIGIOUS_EDUCATION,
+    ],
+  ),
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:opt_in_undergrad],
+    amount: 9_000,
+    allocation_subjects: [
+      AllocationSubjects::ANCIENT_LANGUAGES,
+      AllocationSubjects::COMPUTING,
+      AllocationSubjects::MATHEMATICS,
+      AllocationSubjects::PHYSICS,
+      AllocationSubjects::MODERN_LANGUAGES,
+    ],
+  ),
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:school_direct_tuition_fee],
+    amount: 28_000,
+    allocation_subjects: [
+      AllocationSubjects::CHEMISTRY,
+      AllocationSubjects::COMPUTING,
+      AllocationSubjects::MATHEMATICS,
+      AllocationSubjects::PHYSICS,
+    ],
+  ),
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:school_direct_tuition_fee],
+    amount: 25_000,
+    allocation_subjects: [
+      AllocationSubjects::ANCIENT_LANGUAGES,
+      AllocationSubjects::BIOLOGY,
+      AllocationSubjects::DESIGN_AND_TECHNOLOGY,
+      AllocationSubjects::GEOGRAPHY,
+      AllocationSubjects::MODERN_LANGUAGES,
+    ],
+  ),
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:school_direct_tuition_fee],
+    amount: 10_000,
+    allocation_subjects: [
+      AllocationSubjects::ART_AND_DESIGN,
+      AllocationSubjects::ENGLISH,
+      AllocationSubjects::MUSIC,
+      AllocationSubjects::RELIGIOUS_EDUCATION,
+    ],
+  ),
+].freeze
+
+SCHOLARSHIPS_2024_TO_2025 = [
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:provider_led_postgrad],
+    amount: 30_000,
+    allocation_subjects: [
+      AllocationSubjects::CHEMISTRY,
+      AllocationSubjects::COMPUTING,
+      AllocationSubjects::MATHEMATICS,
+      AllocationSubjects::PHYSICS,
+    ],
+  ),
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:provider_led_postgrad],
+    amount: 27_000,
+    allocation_subjects: [
+      AllocationSubjects::FRENCH_LANGUAGE,
+      AllocationSubjects::GERMAN_LANGUAGE,
+      AllocationSubjects::SPANISH_LANGUAGE,
+    ],
+  ),
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:school_direct_tuition_fee],
+    amount: 30_000,
+    allocation_subjects: [
+      AllocationSubjects::CHEMISTRY,
+      AllocationSubjects::COMPUTING,
+      AllocationSubjects::MATHEMATICS,
+      AllocationSubjects::PHYSICS,
+    ],
+  ),
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:school_direct_tuition_fee],
```

Comment:
Need to double check, as I thought that is getting retired

Reply:
We think this might be needed for legacy trainees. It shouldn't be doing any harm in here if not.